src/languages/dax.js

```diff
@@ -0,0 +1,86 @@
+/*
+Language: Data Analysis Expressions (DAX)
+Author: Stanislaw Swierc <stanislaw.swierc@gmail.com>
+Description: Data Analysis Expressions (DAX) is a library of functions and operators that can be combined to build formulas and expressions in Microsoft SQL Server Analysis Services, Power Pivot in Excel, and Power BI Desktop.
+Website: https://msdn.microsoft.com/en-us/library/ee634217.aspx
+Category: scientific
+*/
+
+function(hljs) {
+  var IDENT_RE = '[a-zA-Z][a-zA-Z0-9._]*';
+
+  return {
+    case_insensitive: true,
+    contains: [
+      hljs.C_LINE_COMMENT_MODE,
+      hljs.C_BLOCK_COMMENT_MODE,
+      {
+        begin: IDENT_RE,
+        lexemes: IDENT_RE,
+        keywords: {
+          keyword:
+            'ABS ACOS ACOSH ADDCOLUMNS ADDMISSINGITEMS ALL ALLEXCEPT ALLNOBLANKROW ' +
```

Comment:
Just wanted to mention that you'll need to have some relevance filled out, so one solution is to add relevancy to your keywords, for instance:
`ABS|0 ACOS|0 ADDCOLUMNS|10 ADDMISSINGITEMS|10 ALL|10 ALLEXCEPT|10 ALLNOBLANKROW|10`

When the Travis CI build fails on your language, you need to be more relevant. When the Travis CI build misdetect another language for your language, you need to be less relevant. It's a good thing you set relevancy to 0 on strings, numbers and identifiers, now you just need to set relevancy to 10 on the keywords, and you're good to go mate. You can see the details of the build process and for what language it failed to detect at by either clicking "Details" or the red X by the change in the pr changelog